### DISCUSSION TITLE
【水博客】VS code Markdown 导出 PDF 时，数学公式未能正确导出 - Muxiner's Blog

### DISCUSSION THREAD

**Discussion by giscus** [Muxiner/muxiner.github.io#34] [博客文章评论]
# 【水博客】VS code Markdown 导出 PDF 时，数学公式未能正确导出 - Muxiner's Blog

解决 Windows 11 系统下 VS code 对 Markdown 文件导出 PDF 时，数学公式未能正确导出的问题。

https://muxiner.github.io/vscode-markdown-pdf-math-latex-error/

**Comment by heyu58** [comment]
补充一下，这种方法似乎会对$$\{\}$$这种公式内大括号的情况出现无法显示的效果。
